MAINTAINERS.md

```diff
@@ -0,0 +1,9 @@
+### Maintainers
+  - Idan Asulin <idan@memphis.dev> [@idanasulinmemphis](https://github.com/idanasulinmemphis)
+  - Avraham Neeman <avraham@memphis.dev> [@avrhamNeeman](https://github.com/avrhamNeeman)
+  - Valera Bronshtein <valera@memphis.dev> [@valeraBr](https://github.com/valeraBr)
+  - Yaniv Ben Hemo <yaniv@memphis.dev> [@yanivbh1](https://github.com/yanivbh1)
+  - Sveta Gimpelson <sveta@memphis.dev> [@SvetaMemphis](https://github.com/SvetaMemphis)
+  - Shay Bratslavsky <shay@memphis.dev> [@shay23b](https://github.com/shay23b)
+  - Or Grinberg <or@memphis.dev> [@orgrMmphs](https://github.com/orgrMmphs)
+  - Shoham Roditi <shoham@memphis.dev> [@shohamroditimemphis](https://github.com/shohamroditimemphis)
```

Comment:
add yourself and delete Or from this file.
Please do the same in all of our repos and open PRs